cli/tsc/99_main_compiler.js

```diff
@@ -100,6 +100,17 @@ delete Object.prototype.__proto__;
     "PermissionStatus",
     "hostname",
     "ppid",
+    "mainModule",
+    "ftruncate",
+    "ftruncateSync",
+    "fdatasync",
+    "fdatasyncSync",
+    "fsync",
+    "fsyncSync",
+    "fstat",
+    "fstatSync",
```

Comment:
fstatSync ?

Reply:
These are unrelated fixes - they are all unstable APIs that were missing from this list so I added them. Can do that in a separate PR too though.

Reply:
Please do